evi check/Form1.cs

```diff
@@ -0,0 +1,650 @@
+                        }
+                        
+                        break;
+
+                    case "GhostOrb":
+                        if (group == 1)
+                        {
+                            this.Evi1GhostOrb.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2GhostOrb.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3GhostOrb.Checked = false;
+                        }
+                        break;
+
+                    case "EMF5":
+                        if (group == 1)
+                        {
+                            this.Evi1EMF5.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2EMF5.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3EMF5.Checked = false;
+                        }
+                        break;
+
+                    case "GhostWriting":
+                        if (group == 1)
+                        {
+                            this.Evi1GhostWriting.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2GhostWriting.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3GhostWriting.Checked = false;
+                        }
+                        break;
+
+                    case "FingerPrints":
+                        if (group == 1)
+                        {
+                            this.Evi1FingerPrints.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2FingerPrints.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3FingerPrints.Checked = false;
+                        }
+                        break;
+
+                    case "SpiritBox":
+                        if (group == 1)
+                        {
+                            this.Evi1SpiritBox.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2SpiritBox.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3SpiritBox.Checked = false;
+                        }
+                       // this.Evi1SpiritBox.Checked = false;
+                        break;
+                }
+            }
```

Comment:
I feel this could be written a bit nicer with a list. I dont know enough c# to get this correct though, but hopefully you'll get the idea:
```suggestion
Dictionary<string, List<System.Windows.Forms.CheckBox>> evinames = new Dictionary<string, List<System.Windows.Forms.CheckBox>>{
{"Cold", {this.Evi1Cold, this.Evi2Cold, this.Evi3Cold}},
{"GhostOrb", {this.Evi1GhostOrb, this.Evi2GhostOrb, this.Evi3GhostOrb}},
{"EMF5", {this.Evi1EMF5, this.Evi2EMF5, this.Evi3EMF5}},
etc...
};
foreach (string button in removechecked)
{
    foreach(KeyValuePair<string, List<System.Windows.Forms.CheckBox>> entry in evinames)
    {
        if(button == entry.key)
        {
            entry.value[group-1].checked = false;
        }           
    }
}
```